.github/workflows/main.yml

```diff
@@ -13,6 +14,7 @@ env:
 jobs:
   docker_images:
     strategy:
+      fail-fast: false
```

Comment:
I think the release process should be atomic and fail-fast should be kept.
If only part of the release is done we would have do re-release and do some manual cleaning which is error prone.

Reply:
it wouldn't really never be atomic, because if a single job is faster to complete before the first failure happened, you end up with a partial release of images.

as example: https://github.com/Alfresco/alfresco-docker-base-java/actions/runs/1865471079

some are green, one is red, the remaining got cancelled in the process

At least with fail fast disabled you got new images for every build and not only the ones that are faster to build than the failing one.

Reply:
yes, but that's the point. I don't think it's a good thing to have "some" images, whatever the reason.
In most cases the push stage should not be reached before all build stages are done (they roughly take the same amount of time). But I agree that when one build takes longer than expected, this is not true anymore.
However, using fail-fast=false will just increase to possiblity for such things to happen and multiply the occurence of us having to do some manual cleanup in case a build didn't go well.
For this reason I think we should keep it as it is.